app/webpack/components/authentication/sign_in.vue

```diff
@@ -167,13 +173,20 @@ export default {
       loading: false,
       showTermsStep: false,
       showVerificationStep: false,
+      showFailureText: false,
+      failureText: 'Sorry, something went wrong! Please try again.',
```

Comment:
Instead of setting a default message here, I recommend we set the message in the `switch` statement later – this way all the possible error messages are in one place. Also, I have a suggestion for the error messages, which I'll post in that section.

Reply:
Done!